jboss/container/openjdk/jdk/openj9-11/module.yaml

```diff
@@ -0,0 +1,32 @@
+schema_version: 1
+
+name: "jboss.container.openjdk.jdk"
+description: "Installs the JDK for OpenJ9 11."
+version: "openj9-11"
+
+labels:
+- name: "org.jboss.product"
+  value: "openjdk"
+- name: "org.jboss.product.version"
+  value: "11"
```

Comment:
@bekionel caught that this should perhaps be 11.0 to match the equivalent openjdk module